pkg/config/config.go

```diff
@@ -562,8 +565,9 @@ func DefaultConfig() (*Config, error) {
 			ImageVolumes:     ImageVolumesMkdir,
 		},
 		NetworkConfig: NetworkConfig{
-			NetworkDir: cniConfigDir,
-			PluginDirs: []string{cniBinDir},
+			CNIDefaultNetwork: "crio",
```

Comment:
I think we should default to empty.

Reply:
Alright, I changed the implementation and the docs to default to `""`